src/Chapter1.hs

```diff
@@ -613,7 +613,7 @@ Implement a function that returns the sum of the last two digits of a number.
 6
 >>> sumLast2 134
 7
->>> sumLast 1
+>>> sumLast2 1
```

Comment:
@arthurborisow Thanks for the fix! Would it be for hard for you to also fix the type of this comment? Just replace

```haskell
{-
=⚔️= Task 9
```

with

```haskell
{- |
=⚔️= Task 9
```

few lines above. Doctest tests only functions inside special comments (with `|` after `{-`), and this part was missing for a comment in this task. That's why we didn't notice this error before.

Reply:
Sure, no problem!

Reply:
@chshersh Another thing that I noticed: `Now it is time to the open pull request with your changes` - `the` part is redundant I believe

Reply:
Yes, nice catch, @arthurborisow! You can add it to this PR as well, if you don't mind 🙂

Reply:
Done